### PR TITLE
Clean up of maintenance guides in prep for translation

### PR DESCRIPTION
**Pull request by krisdale**
Clean up in preparation for translations  …
- Standardize page formatting (use of intro sentence, TOC tree, etc)
- Simplify intro sentences, remove idioms, etc
- Standardize use of roles (remove use of `` and * for emphasis on special words or options)
- Misc clean up (opportunistic)

### CODE REVIEW

File: source/clear-linux/guides/maintenance/architect-lifecycle.rst [hunk 1]
```diff
@@ -3,18 +3,8 @@
...
-
-Maintaining a |CL| derivative requires:
-
-* Monitoring upstream |CL| for new releases
-* Building software packages and staging
-* Employing CI/CD automation for building releases
-* Integrating Quality Assurance for testing and validation
-
-This guide provides the foundation of the recommended infrastructure.
+This guide desribes the basic, recommended infrastructure and workflow for
```
Comment (MCamp859) [inline source/clear-linux/guides/maintenance/architect-lifecycle.rst]: From "desribes" to 'describes"

File: source/clear-linux/guides/maintenance/cpu-performance.rst [hunk 2]
```diff
@@ -69,7 +79,7 @@ voltages based on CPU demand.
 P-states can typically be limited or disabled in a system's firmware (UEFI/BIOS).
 
 Turbo boost
-~~~~~~~~~~~
+-----------
 
 `Turbo Boost technology`_, found on some modern Intel CPUs, allows core(s) on
```
Comment (MCamp859) [inline source/clear-linux/guides/maintenance/cpu-performance.rst]: Branding issue here, should be "Intel® Turbo Boost"

File: source/clear-linux/guides/maintenance/cpu-performance.rst [hunk 3]
```diff
@@ -57,7 +67,7 @@ or completely disabled with :command:`idle=poll`.
 .. _p-states-section:
 
 P-states (performance states)
------------------------------
+=============================
 
 P-states, also known as *SpeedStep* on Intel processors or *Cool'n'Quiet* on
```
Comment (MCamp859) [inline source/clear-linux/guides/maintenance/cpu-performance.rst]: Branding issue here: should be "Intel SpeedStep® technology"

File: source/clear-linux/guides/maintenance/validate-signatures.rst [hunk 4]
```diff
@@ -3,19 +3,28 @@
 Validate signatures
 ###################
 
-|CL-ATTR| offers a way to validate the content of an image or an update. All
+This guide covers how to validate the contents of a |CL-ATTR| image.
```
Comment (MCamp859) [inline source/clear-linux/guides/maintenance/validate-signatures.rst]: change "covers" to "describes"  [more clear for translation]

File: source/clear-linux/guides/maintenance/architect-lifecycle.rst (hunk 1 above)
Comment (krisdale) [reply source/clear-linux/guides/maintenance/architect-lifecycle.rst]: :D

**Comment by krisdale** [comment]
@MCamp859 @mvincerx Requested changes applied.

**Comment by MCamp859** [comment]
LGTM

**Comment by krisdale** [comment]
@mvincerx - @MCamp859 has given her ok.

File: source/clear-linux/guides/maintenance/bulk-provision.rst [hunk 5]
```diff
@@ -3,16 +3,23 @@
...
-files allow us to customize the instance of the installation.
+.. contents::
+   :local:
+   :depth: 1
+
+Overview
+********
+
+To configure a bulk provision, you must:
+
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/bulk-provision.rst]: Just remove  ", you must".

File: source/clear-linux/guides/maintenance/developer-workstation.rst [hunk 6]
```diff
@@ -36,68 +39,67 @@ Table 1 as a starting point.
...
 We recommend that you understand these core concepts in |CL| *before*
 developing your project.
 
-* :ref:`Bundles <bundles-about>`
-* :ref:`Software update <swupd-about>`
-* :ref:`Mixer <mixer-about>`
-* :ref:`Autospec <autospec-about>`
+* :ref:`Software update <swupd-guide>`
+* :ref:`Mixer <mixer>`
+* :ref:`Autospec <autospec>`
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/developer-workstation.rst]: Great catches.

File: source/clear-linux/guides/maintenance/fix-broken-install.rst [hunk 7]
```diff
@@ -40,9 +50,9 @@ Mount root partition, verify, and fix
 
          lsblk
 
-      We'll use `/dev/sda3/` as the root partition example.
+      We'll use /dev/sda3/ as the root partition example.
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/fix-broken-install.rst]: Change to:  `:file:'/dev/sda3/'`

File: source/clear-linux/guides/maintenance/fix-broken-install.rst [hunk 8]
```diff
@@ -40,9 +50,9 @@ Mount root partition, verify, and fix
...
 
-   #. Next, mount the partition to the `/mnt` folder.
+   #. Next, mount the partition to the /mnt folder.
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/fix-broken-install.rst]: Change to: `:file:'mnt/'`

File: source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst [hunk 9]
```diff
@@ -1,25 +1,29 @@
 .. _increase-virtual-disk-size:
 
-Increase virtual disk size of a |CL-ATTR| image
-###############################################
+Increase virtual disk size of a image
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst]: ... an image

File: source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst [hunk 10]
```diff
@@ -77,38 +81,37 @@ Increase virtual disk size
...
-#. Add the |CL| `storage-utils` bundle to install the
+#. Add the :command:`storage-utils` bundle to install the
    :command:`parted` and :command:`resize2fs` tools.
 
    .. code-block:: bash
 
       sudo swupd bundle-add storage-utils
 
-#. Launch the `parted` tool.
+#. Launch the parted tool.
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst]: keep `parted` in tick marks. It's a disk utility.

File: source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst [hunk 11]
```diff
@@ -77,38 +81,37 @@ Increase virtual disk size
...
 
    .. code-block:: bash
 
       sudo parted
 
-#. In the `parted` tool, perform these steps:
+#. In the parted tool, perform these steps:
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst]: Same as above.

**Comment by krisdale** [comment]
@mvincerx I made the requested changes - nice catches :) I did not rebase this branch.

**Comment by mvincerx** [comment]
Thanks for last commit. Hold please. Will rebase and push to this branch.

**Comment by mvincerx** [comment]
LGTM. Rebase done.

**Pull request merged**